src/main/java/com/esri/samples/raster/raster_rendering_rule/README.md

```diff
@@ -0,0 +1,27 @@
+<h1>Raster Rendering Rule</h1>
+
+<p>Create an <code>ImageServiceRaster</code>, fetch the <code>RenderingRules</code> from the service info, and use a <code>RenderingRule</code> to create an <code>ImageServiceRaster</code> and add it to a raster layer. </p>
```

Comment:
This part of the readme is intended as a quick summary of what the user wants to do. Framed as "I want to...":

Suggested edit: "Display a raster on a map and apply different rendering rules to that raster.